boot.rb

```diff
@@ -1,3 +1,12 @@
+require 'dotenv/load'
+
+require 'rollbar'
+if ENV.include?('ROLLBAR_ACCESS_TOKEN')
+  Rollbar.configure do |config|
+    config.access_token = ENV.fetch('ROLLBAR_ACCESS_TOKEN')
```

Comment:
Do we want to have something to pick up ROLLBAR_ENV as well or will that just happen?

Reply:
Hm, good idea, that was in the rails-generated one in transition, but I'd written this first. I'll see if I can add the same thing to be consistent.

Reply:
Added!